implementations/rust/ockam/ockam_api/src/nodes/service.rs

```diff
@@ -221,39 +220,48 @@ pub struct IdentityOverride {
 impl NodeManager {
     async fn access_control(
         &self,
-        resource: &Resource,
-        action: &Action,
+        resource: Resource,
+        action: Action,
         authority: Option<Identifier>,
-        custom_default: Option<&Expr>,
+        expression: Option<Expr>,
     ) -> Result<Arc<dyn IncomingAccessControl>> {
         if let Some(authority) = authority {
             // Populate environment with known attributes:
             let mut env = Env::new();
-
             env.put("resource.id", str(resource.as_str()));
             env.put("action.id", str(action.as_str()));
 
-            // Check if a policy exists for (resource, action) and if not, then
-            // create or use a default entry:
-            if self.cli_state.get_policy(resource, action).await?.is_none() {
-                let fallback = match custom_default {
-                    Some(e) => e.clone(),
-                    None => Expr::CONST_TRUE,
-                };
+            // Build expression
+            let mut expression = expression.unwrap_or(Expr::CONST_TRUE);
+
+            // TODO: verify that this is needed
+            // Add project identifier to the expression, if available
+            if let Ok(project) = self.cli_state.get_node_project(&self.node_name).await {
```

Comment:
Let's not add it for all resources, only for Kafka

Reply:
Done